src/app/shared/components/template/components/combo-box/combo-box.component.ts

```diff
@@ -22,6 +22,7 @@ export class TmplComboBoxComponent
   extends TemplateBaseComponent
   implements ITemplateRowProps, OnInit, OnDestroy
 {
+  public variant: "modal" | "dropdown" | string;
```

Comment:
nit(non-blocking): is there good reason to accept generic `string` type? Looking at the html nothing will displayed in this case